README.md

```diff
@@ -62,36 +123,139 @@ provider:
     region: us-west-2
     security_group:
         # Use IpPermissions to allow SSH access from your working node
+        # Remember to restrict IpRanges for security 
         IpPermissions:
         - FromPort: 22
           ToPort: 22
           IpProtocol: TCP
           IpRanges:
           - CidrIp: 0.0.0.0/0
 ```
-Check example/cluster folder for more Workspace configuration file examples.
+*NOTE:* Remember to change `CidrIp` from `0.0.0.0/0` to restricted IpRanges for TCP port 22 security
+
+Use the following command to create and provision a Workspace:
+
+```
+cloudtik workspace create /path/to/your-workspace-config.yaml
+```
+
+Check `example/cluster` folder for more Workspace configuration file examples.
+
+### 5. Configuring Cloud Storage
+
+If you choose cloud storage as file system or to store stage and event data, cloud storage account is needed.
+
+After follow the guides below for specific cloud provider, you will be able to fill out the corresponding storage filed 
+of your cluster configuration yaml. 
+
+#### AWS
+
+Every object in Amazon S3 is stored in a bucket. Before you can store data in Amazon S3, you must create a bucket.
+Please refer to the S3 [guide](https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-bucket.html) for instructions.
+
+Then fill out the `aws_s3_storage` field.
+
+```
+# Cloud-provider specific configuration.
+provider:
+    type: aws
+    region: us-west-2
+    # S3 configurations for storage
+    aws_s3_storage:
+        s3.bucket: your_s3_bucket
+        s3.access.key.id: your_s3_access_key_id
+        s3.secret.access.key: your_s3_secret_access_key
+
+```
```

Comment:
"Then fill out the `aws_s3_storage` field." -> This is just in the wrong way.
Does user know anything about these configuration? where is the configuration? NOTHING.
So we should not give this here.